src/common/options/rgw.yaml.in

```diff
@@ -3805,3 +3805,34 @@ options:
   flags:
   - startup
   with_legacy: true
+- name: rgw_topic_persistency_time_to_live
+  type: uint
+  level: advanced
+  desc: The rgw retention of persistent topics by time (seconds)
+  default: 4294967295
+  services:
+  - rgw
+  flags:
+  - startup
+  with_legacy: true
+- name: rgw_topic_persistency_max_retries
+  type: uint
+  level: advanced
+  desc: The rgw halting of persistent topics exceeding the max retries
+  default: 4294967295
```

Comment:
same as above. please set to zero, and explain in description that when zero retries are set, we would retry indefinitly